examples/micro_batching/micro_batching_handler.py

```diff
@@ -0,0 +1,34 @@
+import json
+import logging
+from pathlib import Path
+
+from micro_batching import MicroBatching
+
+from ts.torch_handler.image_classifier import ImageClassifier
+
+logger = logging.getLogger(__name__)
+
+
+class MicroBatchingHandler(ImageClassifier):
+    def __init__(self):
+        mb_handle = MicroBatching(self)
+        self.handle = mb_handle
+
+    def initialize(self, ctx):
+        super().initialize(ctx)
+
+        config_file = Path(ctx.system_properties["model_dir"], "micro_batching.json")
```

Comment:
can we use the new feature [model-config.yaml ](https://github.com/pytorch/serve/tree/master/model-archiver#config-file)to set these parameters?

all of the parameters can be accessed via [ctx. model_yaml_config](https://github.com/pytorch/serve/blob/master/docs/configuration.md#config-model)